.github/workflows/master.yml

```diff
@@ -135,7 +135,7 @@ jobs:
       COVERAGE_OUTPUT_DIR: ${{ secrets.COVERAGE_OUTPUT_DIR }}
     name: generate_coverage_data
     needs: prerequisites
-    runs-on: pulumi-ubuntu-8core
+    runs-on: ubuntu-latest
```

Comment:
This change reduces the runner size to a smaller machine. Is this what we want?

Reply:
Yes, `generate_coverage_data` was mistakenly given the larger runner instead of `prerequisites` for the master workflow.